lib/components/networkCard.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import '../models/networkDevice.dart';
+
+class NetworkCard extends StatelessWidget {
+  final NetworkDevice networkDevice;
+
+  NetworkCard(this.networkDevice);
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      child: Text(
+          "IP: " + networkDevice.getIP() + "\nMac: " + networkDevice.getMAC()),
+      margin: EdgeInsets.symmetric(vertical: 3.0, horizontal: 5.0),
+    );
+  }
+}
+
+/* class NetworkCard extends StatefulWidget {
```

Comment:
I think you can delete the commented stuff since you are going with a stateless widget